src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -21,7 +23,8 @@ public class Gripper extends SubsystemBase {
     private static Gripper INSTANCE;
     private final GripperIO io;
     private final GripperInputsAutoLogged inputs = GripperIO.inputs;
-    private final ElevatorInputsAutoLogged elevatorInputs = ElevatorIO.inputs;
+    private final ElevatorInputsAutoLogged elevatorInputs =
+            ElevatorIO.inputs; // TODO: change to supplier
```

Comment:
yis yis suplir is gud

Reply:
63620fc

Reply:
You can use a method reference instead:
```java
Gripper.initialize(gripperIO, elevator::getCarriageHeight);
```
1. Also, remember we said we might want to compensate for the acceleration of subsystems on each other. It might be beneficial to pass these suppliers after subsystems are instantiated (otherwise you'll have a circular dependency between subsystems).
2. We may also want to compensate the drivetrain's acceleration in the gripper's control.

Consult with @vichik123 on both points,

Reply:
1545856

Reply:
Tell me what you decided on the other points.

Reply:
We probably will work on them in the future however @vichik123 does not want to open an issue for them

Reply:
🤔 
#48